lib/auth0/api/v2/users.rb

```diff
@@ -248,11 +248,22 @@ def get_enrollments(user_id)
         # @see https://auth0.com/docs/api/management/v2#!/Users/get_permissions
         #
         # @param user_id [string] The user_id of the permissions to get.
+        # @param options [hash] A hash of options for getting permissions
+        #   - per_page: Number or permissions to return.
+        #   - page: Page number to return, zero-based
+        #   - include_totals: True to include query summary in the result, false or nil otherwise.
```

Comment:
I'd recommend mentioning in the docs here that each of these properties are optional.

What happens on the server if for example the developer supplies one of these properties but not the others? Will they receive an error? If I only give it `page` but not `per_page`, will it assume defaults for `per_page` and if so should we mention those defaults here (or link to something)?

Reply:
optional added, the pagination implementation follows other pagination implementations on this library, if you do not include the optional fields, it should default on the api